requirements.txt

```diff
@@ -1,3 +1,3 @@
-# viktor==13.7.0
+viktor==13.7.0
```

Comment:
fetching the viktor SDK from pypi is not officially supported yet, but it works, nice!